feat: DOC-283: Add gallery templates from docs

caitlinwheeless opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/13766671059)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- 
<!-- Follow Merge -->


We are working to better align the templates that are available in the documentation with those that are available when browsing the template gallery in the UI. 

This PR adds 3 new templates from the docs to the UI:

* [Content Moderation](https://labelstud.io/templates/content_moderation)
* [Medical Imaging Classification with Bounding Boxes](https://labelstud.io/templates/medical_imaging_classification)
* [LLM Response Grading](https://labelstud.io/templates/llm_response_grading)

Also fixes a very small typo in one of our existing templates

caitlinwheeless committed:
feat: DOC-283: Add gallery templates from docs

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13728503985)

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-new-templates-march'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13728503985

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13728503985

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13730008688)

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13730008688

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13766671059)

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-new-templates-march'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13766671059

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13766671059
